plugin/src/main/java/net/dzikoysk/funnyguilds/guild/Guild.java

```diff
@@ -112,6 +114,7 @@ public GuildRank getRank() {
         return this.rank;
     }
 
+    @Deprecated
     public boolean isRanked() {
         PluginConfiguration config = FunnyGuilds.getInstance().getPluginConfiguration();
```

Comment:
Zamiast powtarzać kod niech może to po prostu się odwołuje do tej metody przeniesionej do RankManager'a